### ISSUE TITLE
webpack package still offers a choice of CLI when there's only one

### ISSUE THREAD

**Issue by nickmccurdy** [webpack/webpack#9951]
Previously when both `webpack-cli` and `webpack-command` were supported, trying to use the `webpack` package from the CLI directly would result in a prompt asking which you want to install. However, now that `webpack-command` is no longer supported, this UI is a little inconvenient/confusing in the way it asks if `webpack-cli` should be used.

```
$ webpack
One CLI for webpack must be installed. These are recommended choices, delivered as separate packages:
 - webpack-cli (https://github.com/webpack/webpack-cli)
   The original webpack full-featured CLI.
We will use "yarn" to install the CLI via "yarn add -D".
Do you want to install 'webpack-cli' (yes/no): 
```

Personally I think that since there's no longer a decision for the user to make here, `webpack-cli` should just be installed automatically (or in a non-interactive session, an error message about manually installing should appear). Alternatively, we could ship `webpack-cli` with `webpack` and skip this entirely.

**Comment by sokra** [comment]
> `webpack-cli` should just be installed automatically

We intentionally ask the user, before making modifications to their package.json and installing random dependencies.

> Alternatively, we could ship `webpack-cli` with `webpack` and skip this entirely.

This has not be done, because it increases install size (a lot) even for users not using the CLI. Like when being part of something else (i. e. create-react-app)

**Comment by nickmccurdy** [comment]
In that case, would you like me to reword the message to be less confusing?

**Comment by nickmccurdy** [comment]
It seems like this was fixed by 8017ddee21df9af97e737e3543ffa9c49d62a6ab and released as `v5.0.0-alpha.12`, I just missed it before I'm on `latest` stable.
